src/index.html

```diff
@@ -1,17 +1,158 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+      autocomplete="off"
+    >
+      <fieldset >
+        <legend>Personal information</legend>
+        <label class="form-field ">
+          Surname:
+          <input
```

Comment:
Text fields should have autocomplete="off"

Reply:
isn't fixed

Reply:
I added autocomplete="off" to the <form> element.